server/@types/bapv.d.ts

```diff
@@ -247,3 +242,54 @@ export type VisitsPageSlot = {
 }
 
 export type FlashData = Record<string, string[] | Record<string, string | string[]>[]>
+
+export type PrisonerDetails = {
+  offenderNo: string
+  name: string
+  dob: string
+  convictedStatus: 'Convicted' | 'Remand'
+  category: string
+  location: string
+  prisonName: string
+  incentiveLevel: string
+  visitBalances: VisitBalances
+}
+
+export type visitItem = [
```

Comment:
Would be better as `VisitItem` to keep with case convention on other types.

(I'm assuming this replaces `UpcomingVisitItem` and `PastVisitItem` so those will eventually go? I wonder whether there might be a more compact way to specify this type 🤔 - fine for now though!)